src/pages/index.js

```diff
@@ -5,6 +5,7 @@ import fireDb from '@utilities/firebase'
 import { serialize } from '@utilities/format'
 import FAQExpandable from '@components/faqTemplates/ExpandableWithCategories'
 import About from '@components/about/TwoColumnsAbout'
+import Video from '@components/video/Video'
 
 export default function Index({ flags: { faqFlag }, faq, faqConfig, example, about }) {
```

Comment:
Do you think we can refactor this object into a single useState hook?

Reply:
Can you elaborate on this idea?